networking_oneview/conf/oneview.py

```diff
@@ -38,9 +38,22 @@
     cfg.IntOpt('ov_refresh_interval',
                default=3600,
                help='Interval between periodic task executions in seconds.'),
-    cfg.BoolOpt('developer_mode',
-                default=False,
-                help='Only set this option as true if under developer mode.')
+    cfg.IntOpt('retries_to_lock_sh',
```

Comment:
this 4 parameters its better in the DEFAULT section

Reply:
agreed... but we need some further work to enable it, then move them together with sync interval and mappings, right?
I guess we can leave them here for now and open a new issue with it!

Reply:
create a branch (bug/5328), you can rebase with it and put those in the right section.